views/ctltools/forms/add-ctl-tool.hbs

```diff
@@ -29,17 +28,37 @@
 
 <h3>Medium</h3>
 <hr/>
+<div class="form-group">
+    <label class="control-label">Mit einem externen Medium verknüpfen?</label>
+    <label  style="width: 100%">
+        <input type="checkbox" id="hasMedium" name="hasMedium" value="1" class="form-control" style="width:10%;display:inline-block;">
+        Ja, es soll verknüpft werden
+    </label>
+</div>
+<div class="form-group" style="display: flex; flex-direction: column;">
+    <label class="control-label">Medien-Katalog</label>
+    <div style="display: flex; align-items: center; gap: 10px;">
+        <select name="medium[mediaSourceId]" id="mediaSource" style="flex: 70%; max-width: 70%;">
+            {{#each mediaSources}}
+                <option value="{{this.sourceId}}" data-media-format="{{this.format}}">{{this.label}}</option>
+            {{/each}}
+        </select>
+        <button type="button" class="btn btn-secondary ml-1" id="btn-load-media-metadata" style="flex: 30%; max-width: 30%;">
+            Metadaten laden
+        </button>
+    </div>
+</div>
 <div class="form-group">
     <label class="control-label" for="mediumId">Medium-Id</label>
-    <input type="text" name="medium[mediumId]" id="mediumId" class="form-control" placeholder="urn:bilo:medium:123456789">
+    <input type="text" name="medium[mediumId]" id="mediumId" class="form-control">
 </div>
 <div class="form-group">
-    <label class="control-label" for="mediaSourceId">Medienkatalog-Id</label>
-    <input type="text" name="medium[mediaSourceId]" id="mediaSourceId" class="form-control">
-</div><div class="form-group">
     <label class="control-label" for="publisher">Anbieter</label>
     <input type="text" name="medium[publisher]" id="publisher" class="form-control">
 </div>
+<div class="form-group">
+    <input type="hidden" name="medium[modifiedAt]" id="modifiedAt" class="form-control">
```

Comment:
we should not include modifiedAt yet, since its purpose and use are not fully planned